woocommerce/woocommerce
# New CFE workflow notifications

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:
This PR adds notifications as part of the new CFE workflow.








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Please disable some common workflows in your forked repo, like CI, etc, to save time.

1. Fork this repo in a different org, make sure the name is `woocommerce`.
2. Add this as a remote in your local.
3. Create a new branch `release/9.4` out of `new-cfe-workflow-github-template` and push that upstream in your new repo.
4. Change the slack token and setup your own slack channel, and update the following values.
5.  slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
      slack-channel: ${{ secrets.WOO_CORE_RELESES_DAILY_SLACK_CHANNEL }}
6. Create an issue with CFE template.
7. On that issue, apply label "CFE Approved", you should receive a notification on slack about CFE being approved, 
8. On any issue apply label "CFE Rejected", you should receive a notification on slack.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
